src/view/InputView.js

```diff
@@ -0,0 +1,10 @@
+import { Console } from '@woowacourse/mission-utils';
+
+const InputView = {
+  async readUserInput(message) {
+    const input = await Console.readLineAsync(message);
+    return input;
+  },
```

Comment:
별건 아니지만 간단하게 적어줄 수 있을 것 같아요.
해당 경우에서는 `async` 지시자는 사용하지만 return문에 `await`는 붙이지 않습니다.

\+ 해당 내용을 더 찾아봤는데 [return await, return no await의 차이](https://velog.io/@lemontech119/return-await-VS-no-return-await)가 있네요, 해당 부분도 학습해보시면 좋을 것 같아요

```suggestion
  async readUserInput(message) {
    return Console.readLineAsync(message);
  },
```

Reply:
좋은 의견 감사합니다 😀